tests/report_generator/training/test_training_parser.py

```
@@ -472,13 +472,56 @@ def test_can_parse_requires_nonempty_tb_dir(tmp_path):
+            "experiment/configs/ConfigContainer.yaml",
+            "{}",
+            id="bridge-empty-object",
+        ),
+        pytest.param(
+            MegatronBridgeParser(),
+            "experiment/tb_logs",
+            "experiment/configs/ConfigContainer.yaml",
+            "key: [",
+            id="bridge-malformed",
+        ),
+    ),
```

Ruff (0.15.21)

[warning] Wrong values type in `pytest.mark.parametrize` expected `list` of `tuple`

Use `list` of `tuple` for parameter values

(PT007)